consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/IbftStateMachine.java

```diff
@@ -27,7 +27,7 @@ public IbftStateMachine(final IbftBlockCreatorFactory blockCreatorFactory) {
   /**
    * Attempt to consume the event and update the maintained state
    *
-   * @param event the external action that has occurred
+   * @param event the external Action that has occurred
```

Comment:
This got capitalised in an auto refactor from the Action type?

Reply:
done.